src/Build/GlobalAssemblyInfo.fs

```diff
@@ -10,7 +9,5 @@ open Orleans.CodeGeneration
 [<assembly: AssemblyFileVersion("1.1.0.0")>]
 [<assembly: AssemblyInformationalVersion("1.1.0.0")>]
 
-// generate Orleans serializers for types in FSharp.core.dll
-[<assembly: KnownAssembly(typedefof<unit option>)>]
```

Comment:
This has been moved to assembly-specific AssemblyInfo.fs, because not all F# projects reference Orleans,